packages/components/progress-stepper/examples/ProgressStepperInteractiveExample.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { ProgressStepper } from '@contentful/f36-progress-stepper';
+import { Button, Flex } from '@contentful/f36-components';
+
+export default function ProgressStepperInteractiveExample() {
+  const [currentStep, setCurrentStep] = useState(1);
+
+  const steps = [
+    { title: 'Step 1' },
+    { title: 'Step 2' },
+    { title: 'Step 3' },
+    { title: 'Step 4' },
+    { title: 'Step 5' },
+  ];
+
+  const getSteps = () => {
+    return steps.map((step, index) => {
+      const stepProps = {
+        labelText: step.title,
+        variant: 'incomplete',
+      };
+
+      if (index + 1 === currentStep) {
+        stepProps['variant'] = 'active';
+      } else if (index + 1 < currentStep) {
+        stepProps['variant'] = 'complete';
+      }
+
+      return <ProgressStepper.Step key={step.title} {...stepProps} />;
+    });
```

Comment:
We are aware of the CI failure related to this file. Unfortunately, when we take measures to resolve the typescript error, the example file does not render correctly within the website itself (see screenshot). Curious if it i expected that no Typescript related syntax will render on the website? If so, we will just rework this interactive example. 
<img width="1014" alt="Screenshot 2024-06-27 at 2 41 31 PM" src="https://github.com/contentful/forma-36/assets/58186851/1a05b07a-d582-47fd-bcac-9c9117ccc072">

Reply:
Yeah the live examples are vanilla JS only.

Reply:
Thanks for the clarification @denkristoffer! We have refactored this interactive example so that we no longer see these errors.